Sources/Player/UserInterface/ProgressTracker.swift

```diff
@@ -27,6 +27,8 @@ public final class ProgressTracker: ObservableObject {
         }
     }
 
+    @Published public var loaded: Float = .zero
```

Comment:
Could maybe be confused with an loaded state, maybe calling this value `buffer` (or something similar) might be clearer?

Moreover the documentation is missing.